test/unit/utilsTest.js

```diff
@@ -118,4 +105,59 @@ describe('utils', function () {
       })
     })
   })
+
+  describe('checkPrerequisites', function () {
+    describe('when in npm script', function () {
+      beforeEach(function () {
+        utils.checkPrerequisites(callback)
+      })
+
+      it('yields', function () {
+        expect(callback).to.have.been.calledOnce()
+      })
+    })
+  })
+
+  describe('getCommitMessagesSinceLastRelease', function () {
+    describe('something', function () {
```

Comment:
Do we need this additional `describe('something'..` here?

Reply:
was a placeholder, still tests missing from this...